Source/controls/plrctrls.h

```diff
@@ -5,6 +5,7 @@
 #include <cstdint>
 
 #include <SDL.h>
+#include <SDL_gamecontroller.h>

```

Comment:
SDL1 only has joystick support meaning which requires a bit more management of the individual elements that makes up the gamepad. All of that has been abstracted away in general though to if you implemented things in the right layer you shouldn't have issues with that.

```suggestion
#ifndef USE_SDL1
#include "controls/devices/game_controller.h"
#endif
```